### PR TITLE
Add make.ps1 command shortcuts

### LINKED ISSUE #14880: Question: Adding shortcuts to make.cmd; d for dependencies
Would it be benificial to add shortcuts to the make.cmd? 
E.g. `d` for `dependencies`
`c` for `check`
`s` for `checkscript`
...

Sure it isn't necessary but could save a couple hours lifetime (counted up)

### PR DESCRIPTION
**Pull request by fruestueck**
Closes #14880 adding shortcuts to make.ps1 (make.cmd).

### CODE REVIEW

**Comment by reaperrr** [comment]
You mention the shortcuts in https://github.com/fruestueck/OpenRA/blob/0fcb2583fe0b68d49a7315bd657e59f177a3d9ed/make.ps1#L235 and below, too.

**Comment by fruestueck** [comment]
Yes i did, but i would wait for more comments on https://github.com/OpenRA/OpenRA/issues/14880#issuecomment-370261154 before adding some highlighting. Just adding the shurtcut after the command doesn't look appearing. : )

**Comment by reaperrr** [comment]
I'd go with 
> Write the shortcut next to the command

**Comment by fruestueck** [comment]
This adds 4 characters per line. Is it too big?
My cmd-window has more space per line.

**Comment by reaperrr** [comment]
> My cmd-window has more space per line so i can't really tell.

Same here. If it's that way for all or at least most Windows-users, it should be fine.

**Review by reaperrr — DISMISSED** [review]

File: make.ps1 [hunk 1]
```diff
@@ -232,16 +232,16 @@ if ($args.Length -eq 0)
...
-	echo "                  version for the current Git branch."
-	echo "  clean           Removes all built and copied files. Use the 'all' and"
-	echo "                  'dependencies' commands to restore removed files."
-	echo "  test            Tests the default mods for errors."
-	echo "  check           Checks .cs files for StyleCop violations."
-	echo "  check-scripts   Checks .lua files for syntax errors."
-	echo "  docs            Generates the trait and Lua API documentation."
+	echo "  all, a              Builds the game and its development tools."
+	echo "  dependencies, d     Copies the game's dependencies into the main game folder."
+	echo "  version, v          Sets the version strings for the default mods to the latest"
```
Comment (RoosterDragon) [inline make.ps1]: image

Can you wrap latest? It's 1 character too long on the default size.

Comment (pchote) [reply make.ps1]: "Sets the version strings for the default mods to the latest" reads weirdly anyway, so how about shortening it to "Sets the version strings for the default mods"?

Comment (fruestueck) [reply make.ps1]: I've just moved the last word into the next line, couse it's a twoliner anyway. Could be changed if still wanted.
Commits squashed

File: make.ps1 [hunk 2]
```diff
@@ -258,14 +258,14 @@ if ($command.Length -gt 1)
...
-	"check-scripts" { Check-Scripts-Command }
-	"docs" { Docs-Command }
+	{"all",           "a"  -contains $_} { All-Command }
+	{"dependencies",  "d"  -contains $_} { Dependencies-Command }
+	{"version",       "v"  -contains $_} { Version-Command }
+	{"clean",         "c"  -contains $_} { Clean-Command }
+	{"test",          "t"  -contains $_} { Test-Command }
+	{"check",         "ck" -contains $_} { Check-Command }
+	{"check-scripts", "cs" -contains $_} { Check-Scripts-Command }
+	{"docs",          "d"  -contains $_} { Docs-Command }
```
Comment (RoosterDragon) [inline make.ps1]: `docs` and `dependencies` both have `d` as the shorthand. I would just remove the shortcut for `docs` - it's short enough.

Comment (fruestueck) [reply make.ps1]: Done. Otherwise we could give it 'o' as a shortcut

**Review by RoosterDragon — APPROVED** [review]

**Comment by fruestueck** [comment]
@RoosterDragon can you add a changelog entry? I'm lacking the necessary rights and knowhow.

**Comment by RoosterDragon** [comment]
[Changelog](https://github.com/OpenRA/OpenRA/wiki/Changelog-(bleed)/_compare/bf511d6d7aa30d69254b945d12bccc180057b9bb)

**Pull request merged**